src/types/classgroup.py

```diff
@@ -1,15 +1,26 @@
 from dataclasses import dataclass
 
-from src.util.ints import int512
+from src.consensus.constants import ConsensusConstants
 from src.util.streamable import Streamable, streamable
 
 
 @dataclass(frozen=True)
 @streamable
 class ClassgroupElement(Streamable):
-    a: int512
-    b: int512
+    data: bytes
```

Comment:
Is the size capped? Can we instead use a fixed size for it? The issue here is that we can get ClassgroupElement objects that are unbounded in size, and it can be used as a DOS vector. This makes most API messages unbounded in size, whereas before they were fixed

Reply:
You can add a new bytes100 type and use that

Reply:
I've been thinking about making it a `bytes100` type, but one slight problem with it is that in tests we have a smaller DISCRIMINANT_SIZE_BITS constant, so the size of `bytes100` won't match the intended size of the classgroup element. Though it can be worked around by allowing a larger size in chiavdf. I will try this approach.

Reply:
Yeah, we had the int512 which was much larger than necessary for the tests, it's a similar thing. We can work around it I think. 
Also, using bytes100 uses 4 less bytes, because a "bytes" field is serialized as [Length][actual bytes].